frontend/src/stores/messages.js

```diff
@@ -1,74 +1,69 @@
 import { ApiRoutes } from '@/utils/constants'
 import ApiService from '@/common/apiService'
 import { defineStore } from 'pinia'
+import { isEmpty } from 'lodash'
 
 export const useMessagesStore = defineStore('messages', {
   namespaced: true,
   state: () => ({
-    messages: null,
+    assistanceRequests: null,
```

Comment:
Agreed! This naming inconsistency is confusing. I chose "assistanceRequests" at the beginning because I followed the naming convention in CRM. We can make this change in the future